### DISCUSSION TITLE
moving mouse to edges in editor sometimes scrolls

### DISCUSSION THREAD

**Discussion by Axilotl17** [ppy/osu#16916] [Q&A] [answered]
sometimes when editing, seemingly at random, the screen will scroll when mouse is moved to the left or right edges, in the direction you moved the mouse.

keep note that in the video I am not holding down any buttons (as far as i know) and that when i leave and rejoin it is back to normal. 
(also i am the one pausing and starting the audio at the end)

https://user-images.githubusercontent.com/78441708/154790882-ab177eb2-6403-4ea3-93d4-959d8b8e9d8c.mp4

[logs.zip](https://github.com/ppy/osu/files/8101607/logs.zip)

I was also wondering:
should I put issues like these in [github.com/ppy/osu/issues](https://github.com/ppy/osu/issues) ? I have a lot of these i have been putting off submitting, and I'm just wondering where i should submit obvious bugs, and I know where to suggest ideas separately.

**Comment by frenzibyte** [comment]
> I was also wondering:
should I put issues like these in [github.com/ppy/osu/issues](https://github.com/ppy/osu/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc) ? I have a lot of these i have been putting off submitting, and I'm just wondering where i should submit obvious bugs, and I know where to suggest ideas separately.

All bugs should be reported as discussion threads in the Q&A section. They will then be escalated to issue threads when deemed to be an actual bug with the game.

**Comment by bdach** [comment]
I have never seen anything like this behaviour for however long I have been on linux. Are you sure this isn't some sort of gesture functionality of the window manager causing scroll inputs to be emitted?

**Reply by Axilotl17** [reply]
It might be although i've never seen it happen outside of osu, and it happened on a different computer before i switched to linux. also, hitting escape doesn't fix it, its actually leaving the editor that does the trick. next time it happens (rarely once/hour, unsure how to trigger it) ill do more thorough testing.

**Reply by Axilotl17** [reply]
I am on windows currently, this is practically a fresh install. running on [2022.1022.0](https://github.com/ppy/osu/releases/tag/2022.1022.0) (using for editor because of #21113) this still happens. 

https://user-images.githubusercontent.com/78441708/202358685-8bd753b4-4658-44ef-bdc5-953a62f37b47.mp4

this also still happens on my linux installation. again, infrequently, but it still occurs. 

[logs.zip](https://github.com/ppy/osu/files/10028083/logs.zip)

**Comment by peppy** [comment, marked as answer]
Probably the same issue as https://github.com/ppy/osu/issues/21260?

**Reply by frenzibyte** [reply]
Let's mark this as a duplicate of that for now. The scrolling point is also at the timeline extents, so that checks out.
